test/Faker/Container/ContainerTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Container;
+
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+final class ContainerTest extends TestCase
+{
+    public function testGetNonExisting()
+    {
+        $container = new Container(['file' => FileExtension::class]);
+        self::assertFalse($container->has('foo'));
+        $this->expectException(NotFoundExceptionInterface::class);
+        $container->get('foo');
+    }
+
+    public function testGetFromString()
+    {
+        $container = new Container(['file' => FileExtension::class]);
+        $object = $container->get('file');
+
+        self::assertInstanceOf(FileExtension::class, $object);
+    }
+
+    public function testGetFromNoClassString()
+    {
+        $container = new Container(['file' => 'this is not a class']);
+        $this->expectException(ContainerExceptionInterface::class);
+        $container->get('file');
+    }
+
+    public function testGetFromCallable()
+    {
+        $container = new Container(['file' => \Closure::fromCallable(function () {
+            return new FileExtension();
+        })]);
+        $object = $container->get('file');
+
+        self::assertInstanceOf(FileExtension::class, $object);
+    }
+
+    public function testGetFromObject()
+    {
+        $container = new Container(['file' => new FileExtension()]);
+        $object = $container->get('file');
+
+        self::assertInstanceOf(FileExtension::class, $object);
+    }
+
+    public function testGetFromNull()
+    {
+        $container = new Container(['file' => null]);
+        $this->expectException(ContainerExceptionInterface::class);
+        $container->get('file');
```

Comment:
```suggestion
        $container = new Container(['file' => null]);
        
        $this->expectException(ContainerExceptionInterface::class);
        
        $container->get('file');
```

How do you feel about adding more vertical whitespace to tests?

I think it is generally a good idea to segment statements into the three As (_Arrange_, _Act_, _Assert_). Each segment is like a new paragraph. Vertical whitespace is free!

Reply:
For smaller test, I dont really care. ¯\_(ツ)_/¯ 

Im happy to add vertical whitespace for you.